cmd/kubectl-gadget/deploy.go

```diff
@@ -241,6 +242,9 @@ func init() {
 	deployCmd.PersistentFlags().StringVarP(
 		&gadgetsPublicKey,
 		"gadgets-public-key", "", resources.InspektorGadgetPublicKey, "Public key used to verify the gadgets")
+	deployCmd.PersistentFlags().StringSliceVar(
+		&allowedDigests,
+		"allowed-digests", []string{}, "List of allowed digests, if image digest is not part of it, execution will be denied")
```

Comment:
Add `By default, all digests are allowed` to the end of the description to keep it in sync with the `oci` parameter